VERSION

```diff
@@ -0,0 +1 @@
+1.0.0-pre1
```

Comment:
I don't understand - when would we tag a version on `vector-datasource` that was different from the semver version that the document is describing? If there's a change in the API, for example something in tilequeue or some upstream rewrite rule, then it's possible we'd still want to update documentation in this repo.

If the version would change and this repo would stay entirely identical (apart from `VERSION`), then does that indicate that the `VERSION` file belongs in a different repo, and it's that other repo which is being versioned?